js/apps/system/_admin/aardvark/APP/react/src/views/views/editView/CopyPropertiesDropdown.tsx

```diff
@@ -78,7 +78,7 @@ const CopyPropertiesInner = ({ views }: { views: SearchViewType[] }) => {
             selectedView
           });
           // TODO: remove this once there are forms in search-alias view
-          if (initialView.type === "search-alias") {
+          if ((initialView.type as any) === "search-alias") {
```

Comment:
Don't think this `as any` is required
```suggestion
          if (initialView.type === "search-alias") {
```